.github/workflows/self_test.yaml

```diff
@@ -232,8 +232,8 @@ jobs:
             - name: infrastructure-agent
               to: v1.2.3
             - name: github.com/spf13/viper
-              from: 4.0.3
-              to: 4.1.2
+              from: 1.14.0
+              to: 1.15.0
```

Comment:
Using an existing tag works as expected... but I'm not sure if relying on the external tag to perform self-tests is a good idea. Since we are already covering the Github links in the integration tests, maybe it would make sense to remove this dependency from here. WDYT?

Reply:
Thanks, done here: [dc67e07](https://github.com/newrelic/release-toolkit/pull/134/commits/dc67e07f5c7412344599d6161c6708350dfc70a5)